content/concepts/modules.md

```diff
@@ -18,8 +18,8 @@ webpack builds on lessons learned from these systems and applies the concept of
 
 In contrast to Node.js modules, webpack _modules_ can express their _dependencies_ in a variety of ways. A few examples are:
 
-* A JavaScript `require()` statement
 * An ECMAScript2015 `import` statement
```

Comment:
ECMAScript2015 or ES2015? Maybe we can stick with the latter convention across the docs. Or go with ES6 even.